linkerd/proxy/balance/src/discover/buffer.rs

```diff
@@ -16,48 +16,77 @@ where
 {
     let (tx, rx) = mpsc::channel(capacity);
 
+    let send = |tx: &mpsc::Sender<_>, change| {
```

Comment:
nit, not a big deal: it would be nice if there was a comment here saying what the bool returned by this closure means? something like:
```suggestion
    // Attempts to send an update to the balancer, returning `true` if we should continue
    // processing discovery updates, or `false` if the balancer is dead and this task should 
    // terminate.
    let send = |tx: &mpsc::Sender<_>, change| {
```

Reply:
also, not a big deal, but it seems like this could maybe be a free function rather a closure, since AFAICT it doesn't actually close over anything in scope here?  the closure might result in more code generation than a free function...but this probably doesn't really matter.

Reply:
The free function is needlessly verbose. Biasing for legibility.